content/docs/guides/partner-billing.md

```diff
@@ -0,0 +1,350 @@
+|---------------------------|---------------------------------|----------------------------------|
+| logical_size_bytes      | 100,000,000 (approx. 100 MiB)   | 10,000,000,000 (approx. 10 GiB)  |
+
+
+### Guidelines
+
+Generally, the most effective quotas for controlling spend per project are those controlling maximum compute (`active_time_seconds` and `compute_time_seconds`) and maximum written storage (`written_data_bytes`). In practice, it is possible that `data_transfer_bytes` could introduce unintended logical constraints against your usage. For example, let's say you want to run a cleanup operation to reduce your storage. If part of this cleanup operation involves moving data across the network (for instance, to create an offsite backup before deletion), the `data_transfer_bytes` limit could prevent you from completing the operation &#8212; an undesirable situation where two measures meant to control cost interfere with one another.
+
+## Suspending active computes
+
+_**What happens when the quota is met?**_
+
+When any configured metric reaches its quota limit, all active computes for that project are automatically suspended. It is important to understand, this suspension is persistent. It works differently than the inactivity-based [autosuspend](/docs/guides/auto-suspend-guide), where computes restart at the next interaction: this suspend will _not_ restart at the next API call or incoming proxy connection. If you don't take explicit action otherwise, the suspension remains in place until the end of the current billing period starts (`consumption_period_end`).
+
+See [Querying metrics and quotas](#querying-metrics-and-quotas) to find your reset date, billing period, and other values related to the project's consumption.
+
+<Admonition type="Note">
+Neon tracks these consumption metrics on a monthly cycle. If you want to track metrics on a specific time range, you need to store your metrics as snapshots. You can also use the Preview [Consumption API](#retrieving-metrics-for-all-projects) to collect metrics from across a range of billing periods. 
+</Admonition>
+
+## Configuring quotas
+
+You can set quotas using the Neon API either in a `POST` when you create a project or a `PATCH` to update an existing project:
+* [Set quotas when you create the project](#set-quotas-when-you-create-the-project)
+* [Update an existing project](#update-an-existing-project)
+
+### Set quotas when you create the project
+For performance reasons, you might want to configure these quotas at the same time that you create a new project for your user, reducing the number of API calls you need to make.
+
+Here is a sample `POST` in `curl` that creates a new project called `UserNew` and sets the `active_time_seconds` quota to a total allowed time of 10 hours (36,000 seconds) for the month, and a total allowed `compute_time_seconds` set to 2.5 hours (9,000 seconds) for the month.  
```

Comment:
2.5 hours makes sense if the compute is a .25 CU compute. Should we mention or does it complicate matters?

Reply:
I think it's good to include. How about this as an additional sentence?

"This 4:1 ratio between active and compute time is suitable for a fixed compute size of 0.25 vCPU."